src/sui-link/index.jsx

```diff
@@ -24,7 +25,12 @@ export default class Link extends React.Component{
     );
 
     return (
-        <a className={customclass} href={this.props.url} title={this.props.title} target={this.props.target}>
+        <a className={customclass}
```

Comment:
IMHO It's better all properties inline, the html returned would be a little weird with a link splitted in 4 lines.